packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/form-schema/form-fields.ts

```diff
@@ -0,0 +1,112 @@
+import { FormFieldInfo } from '@reapit/elements'
+
+export type FieldKeys =
+  | 'vatNumberField'
+  | 'noVatNumberField'
+  | 'nationalInsuranceNumberField'
+  | 'officeEmailField'
+  | 'companyNameField'
+  | 'telField'
+  | 'websiteField'
+  | 'regField'
+  | 'noRegField'
+  | 'aboutField'
+  | 'iconImageUrlField'
+  | 'line1Field'
+  | 'line2Field'
+  | 'line3Field'
+  | 'line4Field'
+  | 'countryIdField'
+  | 'postCodeField'
+export type FormFields = Partial<Record<FieldKeys, string>>
+
+export type OrganisationFormValues = {
+  vatNumber: string
+  noVatNumber: boolean
+  nationalInsuranceNumber: string
+  officeEmail: string
+  about: string
+  companyName: string
+  countryId: string
+  iconImageUrl: string
+  line1: string
+  line2: string
+  line3: string
+  line4: string
+  postCode: string
+  reg: string
+  noReg: boolean
+  tel: string
+  website: string
+}
+
+export const formFields: Record<FieldKeys, FormFieldInfo> = {
+  vatNumberField: {
+    name: 'vatNumber',
+    label: 'VAT Number',
+  },
+  noVatNumberField: {
+    name: 'noVatNumber',
+    label: 'No VAT Number',
+  },
+  nationalInsuranceNumberField: {
+    name: 'nationalInsuranceNumber',
+    label: 'National Insurance Number',
+  },
+  officeEmailField: {
+    name: 'officeEmail',
+    label: 'Office Email',
+  },
+  aboutField: {
+    name: 'about',
+    label: 'About',
+  },
+  companyNameField: {
+    name: 'companyName',
+    label: 'Company Name',
+  },
+  countryIdField: {
+    name: 'countryId',
+    label: 'Country',
+  },
+  iconImageUrlField: {
+    name: 'iconImageUrl',
+    label: 'Upload Logo',
+  },
+  line1Field: {
+    name: 'line1',
+    label: 'Line 1*',
```

Comment:
The address fields are now Address Line 1, Address Line 2, Address Line 3 and Address Line 4. Postcode etc remain the same